src/encoded/static/scss/encoded/modules/_chromatin_state.scss

```diff
@@ -380,19 +459,69 @@ $statestroke: #7D6167;
     }
 }
 
+.activeenhancer .overlay {
```

Comment:
The following area would have been perfect for a Sass @each loop — easier to maintain.

Reply:
Yes, that would have saved some typing too